Content.Client/GameObjects/Components/HUD/Inventory/StrippableBoundUserInterface.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using Content.Client.UserInterface;
+using Content.Shared.GameObjects.Components.GUI;
+using Content.Shared.GameObjects.Components.Inventory;
+using JetBrains.Annotations;
+using Robust.Client.GameObjects.Components.UserInterface;
+using Robust.Shared.GameObjects.Components.UserInterface;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Client.GameObjects.Components.HUD.Inventory
+{
+    public class StrippableBoundUserInterface : BoundUserInterface
+    {
+        public Dictionary<EquipmentSlotDefines.Slots, string> Inventory { get; private set; }
+        public Dictionary<string, string> Hands { get; private set; }
+
+        [ViewVariables]
+        private StrippingMenu _strippingMenu;
+
+        public StrippableBoundUserInterface([NotNull] ClientUserInterfaceComponent owner, [NotNull] object uiKey) : base(owner, uiKey)
```

Comment:
```suggestion
        public StrippableBoundUserInterface(ClientUserInterfaceComponent owner, object uiKey) : base(owner, uiKey)
```

No other BoundUserInterface uses NotNull annotations there, and there's only 8 other usages of it in the project.